content/post/on-crypto.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Personal stance on 'crypto' and 'web3'"
+date: 2021-11-18T12:33:09+05:30
+lastmod: 2021-11-18T12:33:09+05:30
+
+# You can also close(false) or open(true) something for this content.
+# P.S. comment can only be closed
+comment: true
+toc: true
+---
+
+This post is inspired by a slack conversation with [Ryan Abernathey](https://twitter.com/rabernat/)
+and [Sarah Gibson](https://twitter.com/drsarahlgibson/).
+
+It is very important to me that we don't exist in a world where the Internet is just centralized
+and controlled by a few huge, hypercapitalist players. I've written about ways the open source
+community can help here ([by not tying ourselves to a single provider](https://words.yuvi.in/post/oss-in-the-cloud/)),
+and helped draft the [right to replicate](https://2i2c.org/right-to-replicate/) policy to
+put it into effect at 2i2c. This topic is important to me - I don't want a future where
+step 1 of learning to code is something extremely vendor specific.
+
+Theoretically, this is very well within the scope of the 'web3' movement (quite different from
+the [web 3.0](https://en.wikipedia.org/wiki/Semantic_Web) movement which has mostly died out).
+I would love for that to be the case, but unfortunately I've found almost nothing about the
+'crypto' or 'web3' communities attractive. I'm going to try write down why in this post.
+
+Most of this isn't unique thought - others have written about these with much more citations,
+and deeper analysis than I have. I encourate you to seek those out and read them.
+
+## Extremely toxic community
+
+Right out of the bat, the self-proclaimed community around the monikers 'web3' and 'crypto'
+seems *extremely* toxic, in ways that are generally order of magnitude more
+toxic than most open source toxicity. Absolute massive dealbreaker. The amount of money
+sloshing around there seems to be attracting the kinda folks I don't really want in
+my communities.
+
+## Environmental impact of proof of work
+
+Difficult for me to get past the environmental impact of most proof of work systems. Any
```

Comment:
This NYT article cloud be a nice reference here - https://www.nytimes.com/interactive/2021/09/03/climate/bitcoin-carbon-footprint-electricity.html